### PR TITLE
get rid of bzr

### PR DESCRIPTION
**Pull request by charbeljc**

### CODE REVIEW

File: default_no_bzr.cfg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[buildout]
```
Comment (nbessi) [inline default_no_bzr.cfg]: what is the use of this file?

**Comment by lepistone** [comment]
@charbeljc I think you can remove the `default_no_bzr.cfg`. Then I'm for merging. Thanks!

File: default_no_bzr.cfg (hunk 1 above)
Comment (charbeljc) [inline default_no_bzr.cfg]: @nbessi This was done in an attempt to accelerate the builds. Hum, guess there is no use for this file now, I'm closing this PR.

**Comment by charbeljc** [comment]
@nbessi, @lepistone done

**Comment by lepistone** [comment]
:+1:

**Pull request merged**